### ISSUE TITLE
Ticking block entity - Crash 

### ISSUE THREAD

**Issue by DocCyblade** [refinedmods/refinedstorage#2831]
#### Issue description:

Was reported to me the server I run was not up. Found it has crashed. Looks like the issue this mod. I used MCEdit and removed the block (a controller) the server started this time, but as soon as the user placed another controller in the same spot it crashes. It seems there is some data that needs removed or changed specifically the refinedstorage_nodes.dat. This file looks not to be a text file so I do not know how to open it

Question is how can we get the server to start (remove the offending block and add it back) And what usually causes this issue if you have seen it before.

...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 1.12.2-14.23.5.2847
- Refined Storage: 1.6.16

Does this issue occur on a server? Yes

#### If a (crash)log is relevant for this issue, link it here:

https://gist.github.com/DocCyblade/e2144279725de97494dc6a358a8c05f9

**Comment by DocCyblade** [comment]
FYI: You may want to update your documentation as does not state clearly that this version is not supported only there is no bug fixes and new features. 

The documentation on the wiki on the [FAQ Page](https://refinedmods.com/refined-storage/wiki/faq.html#what-versions-of-minecraft-are-supportedl) says to see [Minecraft version support] (https://refinedmods.com/refined-storage/wiki/minecraft-version-support.html) is the link that you robot posted. Maybe you need to more clearly define what supported means in your wiki. 

If there is a better forum for my question please point me in that direction.

**Comment by DocCyblade** [comment]
For the sake of anyone else doing a search for the same issue I found I will post my findings here since v1.6.16 is not supported any more.

 The issue was we had over 500 1k storage blocks connected to the controller. Using MCEdit removed the controller block and the server would start. My friend Kenton thought it had to do with the 500 1k storage blocks that was added. We removed the cable from that and put the controller back in, no issues. We put the cable back and the game started throwing the exceptions. Just for testing I will load up FTB DW 1.16 mod pack that and see if I can replicate this error on the new code.
